packages/components/src/components/ScalarCopyButton/ScalarCopyButton.vue

```diff
@@ -0,0 +1,101 @@
+<script setup lang="ts">
+import { ScalarIconCheck, ScalarIconCopy } from '@scalar/icons'
+import { cva, cx } from '@scalar/use-hooks/useBindCx'
+import { useClipboard } from '@scalar/use-hooks/useClipboard'
+import { computed, ref } from 'vue'
+
+import { ScalarButton } from '../ScalarButton'
+
+const props = withDefaults(
```

Comment:
Vue encourages using [props destructuring](https://vuejs.org/guide/components/props#reactive-props-destructure) now instead of `withDefault`

Reply:
good catch!